SPECS/overlays/20overlays/module-setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+check() {
+    return 0
+}
+
+depends() {
+    return 0
+}
+
+install() {
+    inst_hook pre-mount 99 "$moddir/overlays-mount.sh"
```

Comment:
I wonder how this may interact with other modules that setup overlays as well in terms of order.
The dmsquash-live-root setup overlays too... The lower is the rootfs, and the upper is the entire partition.